app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert split_integer(6, 2) == [3, 3]
+    assert split_integer(41, 8) == [5, 5, 5, 5, 5, 5, 5, 6]
```

Comment:
u should to check a sum or parts of result of function, so use sum function here